index.js

```diff
@@ -1 +1,29 @@
-// Do work!
\ No newline at end of file
+function validatePassword(password) {
+  let isSpecial = false
+  let isNumeric = false
+  const stringOfNums = '0123456789'
+
+  if (password.length >= 8) { // if the password length is 8 or more then do code after
+    let upperCase = password.toUpperCase()
+    let lowerCase = password.toLowerCase()
+
+    if (upperCase === password || lowerCase === password) {
```

Comment:
This looks clever. Are you able to confirm if this is checking correctly.
It reads as though it is checking for an all uppercase or or lowercase password having been passed in.
We need it to validate that there is one upper and one lower letter only. 
Maybe it is solving this but from looking closely that is the curiosity I have.